Userland/Applications/Browser/CookiesTab.gml

```diff
@@ -0,0 +1,17 @@
+@GUI::Widget {
+    name: "cookies_tab"
+    layout: @GUI::VerticalBoxLayout {
+        margins: [4]
+    }
+
+    @GUI::GroupBox {
+        layout: @GUI::VerticalBoxLayout {
+            margins: [6]
+        }
+
+        @GUI::TableView {
+            name: "cookies_tableview"
+            font_size: 12
```

Comment:
Please leave the default `font_size` instead of hard-coding a custom one :)

Reply:
Done :)